DependencyInjection/Configuration.php

```diff
@@ -55,6 +55,8 @@ public function getConfigTreeBuilder(): TreeBuilder
                                 ->cannotBeEmpty()
                                 ->defaultValue('UTF-8')
                             ->end()
+                            ->scalarNode('enabled')
```

Comment:
Lease add default value as in create_attachments_dir_if_not_exists: "->defaultTrue()".